test/utils/bytecode.hpp

```diff
@@ -106,11 +106,21 @@ inline bytecode mul(bytecode a, bytecode b)
     return b + a + OP_MUL;
 }
 
+inline bytecode shr(bytecode v, bytecode sh)
+{
+    return v + sh + OP_SHR;
+}
+
 inline bytecode not_(bytecode a)
 {
     return a + OP_NOT;
 }
 
+inline bytecode and_(bytecode a)
+{
+    return a + OP_NOT;
```

Comment:
Typo?

Reply:
Yes. I added this, but later it remained unused.